2018-09-14/solutions/khromets.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$arr1 = [1,2,3,4,5];
+$arr2 = [2,4,8];
+
+function intersection(array $arr1, array $arr2) {
+	$lessArr = count($arr1) < count($arr2) ? $arr1 : $arr2;
```

Comment:
still use tabs, don't you? :D

Reply:
your coding style doesn't follow PSR, to be specific you don't have space after `foreach`, `if` etc